README

```diff
@@ -0,0 +1,15 @@
+Jako bazę danych potraktowałem pliki .csv żeby ułatwić testowanie aplikacji.
+
+Aplikacja wystawia dwa endpointy. "/receiveMessage" oraz "/subscription". Pierwszy służy do przyjęcia wiadomości w formacie JSON, a drugi służy do subskrybowania i odsubskrybowywania usługi.
+
+Żeby zasubskrybować należy wysłać request POST o przykładowej treści:
+{
+      “sender”: “234100200300”
+
+      “recipient”: “48700800999”
+
+      “message”: “start”
+}
+Żeby odsubskrybować należy wysłać podobną wiadomość o treści "stop".
+
+Wiadomość wysłana na endpoint "/receiveMessage" musi mieć taką samą strukturę, jak wiadomość o subskrypcji.
```

Comment:
W Readme dobrą praktyką jest wpis jak aplikacje powinno się uruchamiać, tak by ktoś bez namysłu mógł wejść zobaczyć `to start app run ...` - uruchomić to i zobaczyć czy działa.